### ISSUE TITLE
Crash with mod Carpenter's Blocks

### ISSUE THREAD

**Issue by quarter2** [rwtema/extrautilities_old_issues#261]
---- Minecraft Crash Report ----
// Why did you do that?

Time: 8/20/14 7:42 PM
Description: Unexpected error

java.lang.ClassCastException: net.minecraft.world.ChunkCache cannot be cast to net.minecraft.world.World
    at com.rwtema.extrautils.block.BlockColor.func_149720_d(BlockColor.java:131)
    at carpentersblocks.renderer.BlockHandlerBase.getBlockColor(BlockHandlerBase.java:671)
    at carpentersblocks.renderer.BlockHandlerBase.setColorAndRender(BlockHandlerBase.java:644)
    at carpentersblocks.renderer.BlockHandlerBase.renderMultiTexturedSide(BlockHandlerBase.java:555)
    at carpentersblocks.renderer.BlockHandlerBase.delegateSideRender(BlockHandlerBase.java:598)
    at carpentersblocks.renderer.BlockHandlerBase.renderBlock(BlockHandlerBase.java:762)
    at carpentersblocks.renderer.BlockHandlerBase.renderBlockWithRotation(BlockHandlerBase.java:164)
    at carpentersblocks.renderer.BlockHandlerCarpentersDoor.renderTypeGlassTop(BlockHandlerCarpentersDoor.java:113)
    at carpentersblocks.renderer.BlockHandlerCarpentersDoor.renderCarpentersBlock(BlockHandlerCarpentersDoor.java:39)
    at carpentersblocks.renderer.BlockHandlerBase.renderWorldBlock(BlockHandlerBase.java:127)
    at cpw.mods.fml.client.registry.RenderingRegistry.renderWorldBlock(RenderingRegistry.java:118)
    at net.minecraft.src.FMLRenderAccessLibrary.renderWorldBlock(FMLRenderAccessLibrary.java:53)
    at net.minecraft.client.renderer.RenderBlocks.func_147805_b(RenderBlocks.java:296)
    at net.minecraft.client.renderer.WorldRenderer.func_147892_a(WorldRenderer.java:181)
    at net.minecraft.client.renderer.RenderGlobal.func_72716_a(RenderGlobal.java:1530)
    at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1187)
    at net.minecraft.client.renderer.EntityRenderer.func_78480_b(EntityRenderer.java:1015)
    at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:990)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:887)
    at net.minecraft.client.main.Main.main(SourceFile:148)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:134)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:273)
    at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:363)
    at org.multimc.EntryPoint.listen(EntryPoint.java:165)
    at org.multimc.EntryPoint.main(EntryPoint.java:54)
## A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Stacktrace:
    at com.rwtema.extrautils.block.BlockColor.func_149720_d(BlockColor.java:131)
    at carpentersblocks.renderer.BlockHandlerBase.getBlockColor(BlockHandlerBase.java:671)
    at carpentersblocks.renderer.BlockHandlerBase.setColorAndRender(BlockHandlerBase.java:644)
    at carpentersblocks.renderer.BlockHandlerBase.renderMultiTexturedSide(BlockHandlerBase.java:555)
    at carpentersblocks.renderer.BlockHandlerBase.delegateSideRender(BlockHandlerBase.java:598)
    at carpentersblocks.renderer.BlockHandlerBase.renderBlock(BlockHandlerBase.java:762)
    at carpentersblocks.renderer.BlockHandlerBase.renderBlockWithRotation(BlockHandlerBase.java:164)
    at carpentersblocks.renderer.BlockHandlerCarpentersDoor.renderTypeGlassTop(BlockHandlerCarpentersDoor.java:113)
    at carpentersblocks.renderer.BlockHandlerCarpentersDoor.renderCarpentersBlock(BlockHandlerCarpentersDoor.java:39)
    at carpentersblocks.renderer.BlockHandlerBase.renderWorldBlock(BlockHandlerBase.java:127)
    at cpw.mods.fml.client.registry.RenderingRegistry.renderWorldBlock(RenderingRegistry.java:118)
    at net.minecraft.src.FMLRenderAccessLibrary.renderWorldBlock(FMLRenderAccessLibrary.java:53)
    at net.minecraft.client.renderer.RenderBlocks.func_147805_b(RenderBlocks.java:296)
    at net.minecraft.client.renderer.WorldRenderer.func_147892_a(WorldRenderer.java:181)
    at net.minecraft.client.renderer.RenderGlobal.func_72716_a(RenderGlobal.java:1530)
    at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1187)

-- Affected level --
Details:
    Level name: MpServer
    All players: 1 total; [EntityClientPlayerMP['quarter'/16, l='MpServer', x=17.26, y=64.62, z=-222.61]]
    Chunk stats: MultiplayerChunkCache: 25, 25
    Level seed: 0
    Level generator: ID 01 - flat, ver 0. Features enabled: false
    Level generator options: 
    Level spawn location: World: (0,63,0), Chunk: (at 0,3,0 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
    Level time: 19303089 game time, 72011700 day time
    Level dimension: 0
    Level storage version: 0x00000 - Unknown?
    Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
    Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
    Forced entities: 2 total; [EntityClientPlayerMP['quarter'/16, l='MpServer', x=17.26, y=64.62, z=-222.61], EntityWaypointContainer['unknown'/145, l='MpServer', x=0.00, y=0.00, z=0.00]]
    Retry entities: 0 total; []
    Server brand: cauldron,craftbukkit,mcpc,fml,forge
    Server type: Non-integrated multiplayer server
Stacktrace:
    at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:373)
    at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2433)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:916)
    at net.minecraft.client.main.Main.main(SourceFile:148)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:134)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:273)
    at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:363)
    at org.multimc.EntryPoint.listen(EntryPoint.java:165)
    at org.multimc.EntryPoint.main(EntryPoint.java:54)

-- System Details --
Details:
    Minecraft Version: 1.7.10
    Operating System: Windows 7 (amd64) version 6.1
    Java Version: 1.7.0_60, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 373273752 bytes (355 MB) / 1152385024 bytes (1099 MB) up to 1152385024 bytes (1099 MB)
    Mod Pack: Unknown / None
    LiteLoader Mods: 1 loaded mod(s)
          - VoxelMap version 1.2.3
    LaunchWrapper: 42 active transformer(s)
          - Transformer: cpw.mods.fml.common.asm.transformers.PatchingTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.MarkerTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.SideTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.EventSubscriptionTransformer
          - Transformer: codechicken.lib.asm.ClassHeirachyManager
          - Transformer: codechicken.core.asm.InterfaceDependancyTransformer
          - Transformer: codechicken.core.asm.TweakTransformer
          - Transformer: codechicken.core.asm.DelegatedTransformer
          - Transformer: codechicken.core.asm.DefaultImplementationTransformer
          - Transformer: cofh.asm.TransformerCore
          - Transformer: cofh.asm.PCCASMTransformer
          - Transformer: invtweaks.forge.asm.ContainerTransformer
          - Transformer: codechicken.nei.asm.NEITransformer
          - Transformer: squeek.spiceoflife.asm.ClassTransformer
          - Transformer: tconstruct.preloader.ASMInterfaceRepair
          - Transformer: tconstruct.preloader.AccessTransformers
          - Transformer: com.mumfrey.liteloader.transformers.event.EventProxyTransformer
          - Transformer: com.mumfrey.liteloader.launch.LiteLoaderTransformer
          - Transformer: com.mumfrey.liteloader.client.transformers.CrashReportTransformer
          - Transformer: openmods.OpenModsClassTransformer
          - Transformer: openblocks.asm.OpenBlocksClassTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.DeobfuscationTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.AccessTransformer
          - Transformer: net.minecraftforge.transformers.ForgeAccessTransformer
          - Transformer: codechicken.core.asm.CodeChickenAccessTransformer
          - Transformer: cofh.asm.PCCAccessTransformer
          - Transformer: invtweaks.forge.asm.ITAccessTransformer
          - Transformer: tonius.simplyjetpacks.core.SJAccessTransformers
          - Transformer: tconstruct.preloader.AccessTransformers
          - Transformer: cpw.mods.fml.common.asm.transformers.ModAccessTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.TerminalTransformer
          - Transformer: com.mumfrey.liteloader.client.transformers.LiteLoaderEventInjectionTransformer
          - Transformer: com.mumfrey.liteloader.client.transformers.LiteLoaderCallbackInjectionTransformer
          - Transformer: com.mumfrey.liteloader.client.transformers.JoinGamePacketTransformer
          - Transformer: com.mumfrey.liteloader.client.transformers.ServerChatPacketTransformer
          - Transformer: com.mumfrey.liteloader.client.transformers.LoginSuccessPacketTransformer
          - Transformer: com.mumfrey.liteloader.client.transformers.ChatPacketTransformer
          - Transformer: com.thevoxelbox.voxelmap.litemod.VoxelMapTransformer
          - Transformer: com.mumfrey.liteloader.client.transformers.ServerCustomPayloadPacketTransformer
          - Transformer: com.mumfrey.liteloader.client.transformers.CustomPayloadPacketTransformer
          - Transformer: com.mumfrey.liteloader.client.transformers.MinecraftOverlayTransformer
          - Transformer: cpw.mods.fml.common.asm.transformers.ModAPITransformer
    JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx1200m -XX:PermSize=156m
    AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
    IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
    FML: MCP v9.05 FML v7.10.25.1206 Minecraft Forge 10.13.0.1206 90 mods loaded, 90 mods active
    mcp{9.05} [Minecraft Coder Pack](minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    FML{7.10.25.1206} [Forge Mod Loader](forge-1.7.10-10.13.0.1206.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    Forge{10.13.0.1206} [Minecraft Forge](forge-1.7.10-10.13.0.1206.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    CodeChickenCore{1.0.3.23} [CodeChicken Core](minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    <CoFH ASM>{000} [CoFH ASM Data Initialization](minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    NotEnoughItems{1.0.3.49} [Not Enough Items](NotEnoughItems-1.7.10-1.0.3.49-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    simplyjetpackscore{1.0} [Simply Jetpacks Coremod](minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    TConstruct-Preloader{0.1.1} [Tinkers Corestruct](minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    OpenModsCore{@VERSION@} [OpenModsCore](minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    BiblioCraft{1.7.5} [BiblioCraft](BiblioCraft[v1.7.5][MC1.7.10].jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    AWWayofTime{v1.0.1g} [Blood Magic: Alchemical Wizardry](Blood Magic-v1.1.0%281.7.10%29.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    BuildCraft|Core{6.0.17} [BuildCraft](buildcraft-6.0.17.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    BuildCraft|Builders{6.0.17} [BC Builders](buildcraft-6.0.17.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    BuildCraft|Energy{6.0.17} [BC Energy](buildcraft-6.0.17.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    BuildCraft|Factory{6.0.17} [BC Factory](buildcraft-6.0.17.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    BuildCraft|Transport{6.0.17} [BC Transport](buildcraft-6.0.17.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    BuildCraft|Silicon{6.0.17} [BC Silicon](buildcraft-6.0.17.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    CarpentersBlocks{3.2.6} [Carpenter's Blocks](Carpenter's Blocks v3.2.6 - MC 1.7.10.zip) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    chisel{1.5.6a} [Chisel](Chisel-1.7.10-1.5.6a.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    CoFHCore{1.7.10R3.0.0B3} [CoFH Core](CoFHCore-[1.7.10]3.0.0B3-26.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    customnpcs{1.7.10b} [CustomNpcs](CustomNPCs_1.7.10b.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    DamageIndicatorsMod{3.2.0} [Damage Indicators](DamageIndicatorsMod-3.2.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    enviromine{1.2.16} [EnviroMine](EnviroMine-v1.2.16.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    exnihilo{1.35b} [Ex Nihilo](Ex-Nihilo-1.35b.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    ForgeMultipart{1.1.0.296} [Forge Multipart](ForgeMultipart-1.7.10-1.1.0.296-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    ExtraUtilities{1.1.0h} [Extra Utilities](extrautilities-1.1.0h.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    IC2{2.2.607-experimental} [IndustrialCraft 2](industrialcraft-2-2.2.607-experimental.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    GraviSuite{1.7.10-2.0.1} [Graviation Suite](GraviSuite-1.7.10-2.0.1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    HardcoreQuesting{The Journey (4.2.0)} [Hardcore Questing Mode](HQM-The Journey %284.2.0%29.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    IC2NuclearControl{1.7.10.02} [Nuclear Control](ic2nc-1.7.10-1.7.10.02.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    iChunUtil{4.0.0} [iChunUtil](iChunUtil-4.0.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    inventorytweaks{1.58-147-645ca10} [Inventory Tweaks](InventoryTweaks-1.58-147.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    IronChest{6.0.43.730} [Iron Chest](ironchest-1.7.10-6.0.43.730-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    Waila{1.5.3} [Waila](Waila-1.5.3_1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    JABBA{1.1.4} [JABBA](Jabba-1.1.4_1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    Mantle{1.7.10-165.4bc3343} [Mantle](Mantle_mc1.7.10_0.3.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded{1.7.10R2.8.0RC2} [MineFactory Reloaded](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatAppliedEnergistics{1.7.10R2.8.0RC2} [MFR Compat: Applied Energistics](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatAtum{1.7.10R2.8.0RC2} [MFR Compat: Atum](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatBackTools{1.7.10R2.8.0RC2} [MFR Compat: BackTools](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatBuildCraft{1.7.10R2.8.0RC2} [MFR Compat: BuildCraft](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatChococraft{1.7.10R2.8.0RC2} [MFR Compat: Chococraft](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatExtraBiomes{1.7.10R2.8.0RC2} [MFR Compat: ExtraBiomes](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatForestry{1.7.10R2.8.0RC2} [MFR Compat: Forestry](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatForgeMicroblock{1.7.10R2.8.0RC2} [MFR Compat: ForgeMicroblock](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatIC2{1.7.10R2.8.0RC2} [MFR Compat: IC2](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatMystcraft{1.7.10R2.8.0RC2} [MFR Compat: Mystcraft](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatPams{1.7.10R2.8.0RC2} [MFR Compat: Pam's Mods](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    ProjRed|Core{4.4.5.44} [ProjectRed](ProjectRed-Base-1.7.10-4.4.5.44.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    ProjRed|Exploration{4.4.5.44} [ProjectRed-Exploration](ProjectRed-World-1.7.10-4.4.5.44.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatProjRed{1.7.10R2.8.0RC2} [MFR Compat ProjectRed](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    Railcraft{9.2.2.0} [Railcraft](Railcraft_1.7.10-9.2.2.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatRailcraft{1.7.10R2.8.0RC2} [MFR Compat: Railcraft](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatRP2{1.7.10R2.8.0RC2} [MFR Compat: RP2](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatSufficientBiomes{1.7.10R2.8.0RC2} [MFR Compat: Sufficient Biomes](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatThaumcraft{1.7.10R2.8.0RC2} [MFR Compat: Thaumcraft](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    ThermalFoundation{1.7.10R1.0.0B1} [Thermal Foundation](ThermalFoundation-[1.7.10]1.0.0B1-6.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    ThermalExpansion{1.7.10R4.0.0B3} [Thermal Expansion](ThermalExpansion-[1.7.10]4.0.0B3-11.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatThermalExpansion{1.7.10R2.8.0RC2} [MFR Compat: Thermal Expansion](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatTwilightForest{1.7.10R2.8.0RC2} [MFR Compat: TwilightForest](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineFactoryReloaded|CompatVanilla{1.7.10R2.8.0RC2} [MFR Compat: Vanilla](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    MineTweaker3{1.7.10-3.0.3} [MineTweaker 3](MineTweaker3-1.7.10-3.0.6.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    ModTweaker{0.5c} [ModTweaker](ModTweaker-1.7.X-0.5c.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    NEIAddons{1.12.0.4} [NEI Addons](neiaddons-mc1710-1.12.0.4.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    NEIAddons|ExtraBees{1.12.0.4} [NEI Addons: Extra Bees](neiaddons-mc1710-1.12.0.4.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    NEIAddons|Forestry{1.12.0.4} [NEI Addons: Forestry](neiaddons-mc1710-1.12.0.4.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    NEIAddons|CraftingTables{1.12.0.4} [NEI Addons: Crafting Tables](neiaddons-mc1710-1.12.0.4.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    NEIAddons|ExNihilo{1.12.0.4} [NEI Addons: Ex Nihilo](neiaddons-mc1710-1.12.0.4.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    NEM-VersionChecker{0.1} [NEM Version Checker](NEM-VersionChecker-1.7.10-1.0 %281%29.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    OpenMods{0.6} [OpenMods](OpenModsLib-1.7.10-0.6-snapshot-224.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    OpenBlocks{1.3} [OpenBlocks](OpenBlocks-1.7.10-1.3-snapshot-431.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    harvestcraft{1.7.2d} [Pam's HarvestCraft](Pam's HarvestCraft 1.7.10a.zip) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    PneumaticCraft{1.2.5} [PneumaticCraft](PneumaticCraft-1.2.5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    PowerConverters{1.7.10-3.0.0.22} [Power Converters](PowerConverters-1.7.10-3.0.0.22.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    ProjRed|Transmission{4.4.5.44} [ProjectRed-Transmission](ProjectRed-Integration-1.7.10-4.4.5.44.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    ProjRed|Transportation{4.4.5.44} [ProjectRed-Transportation](ProjectRed-Mechanical-1.7.10-4.4.5.44.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    TConstruct{1.6.0d37} [Tinkers' Construct](TConstruct_mc1.7.10_1.6.0d38.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    ProjRed|Compatibility{4.4.5.44} [ProjectRed-Compatibility](ProjectRed-Compat-1.7.10-4.4.5.44.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    ProjRed|Integration{4.4.5.44} [ProjectRed-Integration](ProjectRed-Integration-1.7.10-4.4.5.44.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    ProjRed|Illumination{4.4.5.44} [ProjectRed-Illumination](ProjectRed-Lighting-1.7.10-4.4.5.44.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    ProjRed|Expansion{4.4.5.44} [ProjectRed-Expansion](ProjectRed-Mechanical-1.7.10-4.4.5.44.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    simplyjetpacks{1.0.1} [Simply Jetpacks](SimplyJetpacks-MC1.7.10-1.0.1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    SpiceOfLife{1.1.1} [The Spice of Life](SpiceOfLife-mc1.7.x-1.1.1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    StevesFactoryManager{A90} [Steve's Factory Manager](StevesFactoryManagerA90.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    Sync{4.0.0} [Sync](Sync-4.0.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    TiCTooltips{1.1.8a} [TiC Tooltips](TiC-Tooltips-mc1.7.10-1.1.8a.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    VersionChecker{1.1.5} [Version Checker](VersionChecker-1.1.5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    WailaHarvestability{1.0.4} [Waila Harvestability](WailaHarvestability-mc1.7.x-1.0.4.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    McMultipart{1.1.0.296} [Minecraft Multipart Plugin](ForgeMultipart-1.7.10-1.1.0.296-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    ForgeMicroblock{1.1.0.296} [Forge Microblocks](ForgeMultipart-1.7.10-1.1.0.296-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available
    Mantle Environment: Environment healthy.
    TConstruct Environment: Environment healthy.
    Launched Version: 1.7.10
    LWJGL: 2.9.1
    OpenGL: AMD Radeon HD 5700 Series GL version 4.3.12618 Compatibility Profile Context 13.251.0.0, ATI Technologies Inc.
    GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

```
Is Modded: Definitely; Client brand changed to 'fml,forge'
Type: Client (map_client.txt)
Resource Packs: []
Current Language: English (US)
Profiler Position: N/A (disabled)
Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
Anisotropic Filtering: Off (1)
```

**Comment by Schevv** [comment]
Same problem here. Crashes on versions 1.1.0g and 1.1.0h, but works with 1.1.0f
Issue seems to be caused by using colored bricks with carpenter blocks

**Comment by nightstalker6669** [comment]
i'm also getting this crash when placing any of the lamps in carpenters blocks

**Comment by Lanse505** [comment]
This is not a problem with ExtraUtils, this is a rendering issue on Carpenters blocks side of things. This crash happens with Magical crops colored wood also. Carpenters blocks seems to not be able to render certain blocks for whatever reason. Hence this being a problem with Carpenters blocks i would suggest you go to https://github.com/Mineshopper/carpentersblocks/issues.

**Comment by rwtema** [comment]
Should be fixed in 1.1.0i
